p2p_wallet/Scenes/Main/BankTransfer/BankTransferInfoViewModel.swift

```diff
@@ -0,0 +1,75 @@
+import BankTransfer
+import SwiftUI
+import Foundation
+import Resolver
+import KeyAppUI
+
+final class BankTransferInfoViewModel: BaseViewModel, ObservableObject {
+
+    // MARK: - Dependencies
+
+    @Injected private var bankTransferService: BankTransferService
+
+    // MARK: -
+
+    @Published var items: [any Renderable] = []
+    
+    // MARK: -
+
+    override init() {
+        super.init()
+
+        bind()
+    }
+
+    func bind() {
+        bankTransferService.userData.map { _ in
+            if self.bankTransferService.isBankTransferAvailable() && false {
+                return [
+                    BankTransferInfoImageViewCellItem(image: .bankTransferInfoAvailableIcon),
+                    BankTransferTitleCellItem(title: L10n.openIBANAccountForInternationalTransfersWithZeroFees),
```

Comment:
why have not you combined BankTransferInfoImageViewCellItem and BankTransferTitleCellItem into one view?